server/api/users.js

```diff
@@ -6,15 +6,15 @@ const usersDb = require("../services/database/users");
  * The route here will be: /users/ (remember the prefix users is defined in api/index.js)
  */
 router.get("/", (req, res) => {
-  usersDb
-    .getAllUsers()
-    .then((data) => {
-      res.send(data);
-    })
-    .catch((err) => {
-      console.error(err);
-      res.status(500);
-    });
+	usersDb
```

Comment:
Actually fix all files you changed their indentation 😄 just right click and chose format document.